packages/google_maps_flutter/lib/src/google_maps_controller.dart

```diff
@@ -332,17 +357,48 @@ class GoogleMapController {
     }
   }
 
+  LatLng _convertToLatLng(String value) {
+    try {
```

Comment:
or you can use try where you try to cast a type

Reply:
Done.